modules/ROOT/pages/design-create-publish-api-fragment.adoc

```diff
@@ -30,19 +31,16 @@ An API fragment is a RAML document that has a version and an identifier, but is
 == Procedure
 
 . On the *Projects* page in Design Center, click *Create* and select *Create Fragment*.
-. In the *New API Fragment* dialog, type a name for the project and select the type of API fragment that you want to create. Then, click *Create*.
+. In the *New Fragment* dialog, type a name for the project. You can change the name (but not the extension) later, if you want to.
+. Select the type of API fragment that you want to create. Then, click *Create Fragment*.
 +
 _Result:_ The text editor opens. The editor is divided into three panels:
 +
-* The left panel lists the files in your project. By default, the editor creates a `.raml` file that has the same name as the project.
+* The left panel lists the files in your project.
 +
-[TIP]
-====
-
-You can create more than one API specification in a single project. If you want to include additional API specifications in your project, click the plus sign in the Files panel to create a new file or click the dots that are next to the plus sign to import a file.
-
-
-====
+By default, the editor creates a `.raml` file that has the same name as the project. This file is set as the root file of the project.
```

Comment:
Per style reference:
```suggestion
By default, the editor creates a RAML file that has the same name as the project. 
This file is set as the root file of the project.
```
Or, if you need to specify that this is the file extension:
```suggestion
By default, the editor creates a file that has the same name as the project, appended with `.raml`. 
This file is set as the root file of the project.
```